cosa/encoders/coreir.py

```diff
@@ -338,6 +338,8 @@ def extract_value(x, modname, inst_intr, inst_conf, inst_mod):
                     xval = 1 if xval else 0
                 else:
                     if type(xval) != int:
+                        if xval.is_x():
```

Comment:
This part of the code has try-catch sections in order to take into account the different versions of the bit-vector package. Having a call to is_x outside it would brake the compatibility with previous versions. I suggest to move it inside the right try statement below it.